services/github/github-checks-status.tester.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('branch checks - passing')
+  .get('/badges/shields/master.json')
+  .expectBadge({
+    label: 'checks',
+    message: 'passing',
+    color: 'brightgreen',
+  })
+
+t.create('branch checks - failing')
+  .get('/badges/shields/now-fix.json')
+  .expectBadge({
+    label: 'checks',
+    message: 'failing',
+    color: 'red',
+  })
```

Comment:
Having fixed tests like this with a live project has a strong potential for flakiness. For service tests we typically utilize the helpers to ensure that we're both (A) still successfully able to integrate with the upstream endpoint and work with the response and (B) produce a badge that's in the expected shape or form (in this case a build status badge).

If we have non-trivial logic handling rendering concerns or data extraction/transformation then we'll typically utilize unit tests to cover those functions directly, or in some cases mock the api response from the upstream provider so that there's a known/fixed response.

We don't need to do either of those things here (the build helpers are both heavily battle tested and extremely well covered in existing tests), so we can drop one of these and use the standard test validator in a single case

```suggestion
t.create('branch checks (branch)')
  .get('/badges/shields/master.json')
  .expectBadge({
    label: 'checks',
    message: isBuildStatus,
  })
```